charts/nri-prometheus/templates/deployment.yaml

```diff
@@ -1,26 +1,25 @@
-{{- if (include "nri-prometheus.areValuesValid" .) }}
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  name: {{ template "nri-prometheus.fullname" . }}
+  name: {{ template "newrelic.common.naming.fullname" . }}
   namespace: {{ .Release.Namespace }}
   labels:
-{{ include "nri-prometheus.labels" . | indent 4 }}
+    {{- include "newrelic.common.labels" . | nindent 4 }}
 spec:
   replicas: 1
   selector:
     matchLabels:
-      app.kubernetes.io/name: {{ include "nri-prometheus.name" . }}
+      app.kubernetes.io/name: {{ include "newrelic.common.naming.name" . }}
```

Comment:
just found that there is a special template function for this :) I will change it